### DISCUSSION TITLE
Data Importer and empty numeric fields

### DISCUSSION THREAD

**Discussion by djbodya** [pimcore/pimcore#15674] [General]
I am faced interesting behavior.
I am trying to import data from CSV and I am transform value as Numeric. When the field is empty, in a preview window I am getting "-- EMPTY --" value but when proceed the value is "0". Is it a bug? How i can set field to empty when there is no data in CSV?

**Comment by robertSt7** [comment]
Hi @djbodya thanks for reporting this bug. I have created a new issue for this.
